test/unit/test_model.py

```diff
@@ -135,13 +135,23 @@ def test_set_megasas_storage_controller(self):
             backend_storage_info = [{
                 "type": "megasas",
                 "max_drive_per_controller": 6,
+                "use_jbod": True,
+                "use_msi": True,
+                "max_cmds": 1024,
+                "max_sge": 128,
+                "sas_address": "000abc",
                 "drives": [{"size": 8, "file": "/tmp/sda.img"}]
             }]
             storage = model.CBackendStorage(backend_storage_info)
             storage.init()
             storage.precheck()
             storage.handle_parms()
             assert "-device megasas" in storage.get_option()
+            assert "use_jbod=True" in storage.get_option()
+            assert "use_msi=True" in storage.get_option()
+            assert "max_cmds=1024" in storage.get_option()
+            assert "max_sge=128" in storage.get_option()
+            assert "sas_address=000abc" in storage.get_option()
```

Comment:
@xiar please review if these values make sense for corresponding attribute.

Please review below drive unit test's rationality as well.